api/routes_account.go

```diff
@@ -208,3 +208,30 @@ func (api *API) getIPFSKeyNamesForAuthUser(c *gin.Context) {
 		"key_ids":   keys["key_ids"],
 	})
 }
+
+// ChangeEthereumAddress is used to change a user's ethereum address
+func (api *API) changeEthereumAddress(c *gin.Context) {
+	username := GetAuthenticatedUserFromContext(c)
+	ethAddress, exists := c.GetPostForm("eth_address")
+	if !exists {
+		FailNoExistPostForm(c, "eth_address")
+		return
+	}
+	um := models.NewUserManager(api.DBM.DB)
+	_, err := um.ChangeEthereumAddress(username, ethAddress)
+	if err != nil {
```

Comment:
Do you have any thoughts on using this syntax instead for functions where no return values are kept:

```go
if _, err = um.ChangeEthereumAddress( ... ); if err != nil {
    ...
}
```

Reply:
I like this syntax a lot and it is much cleaner, sounds like a good plan to me.